docs/tutorials/experimental-fits.md

```diff
@@ -0,0 +1,436 @@
+# Experimental data fits
+
+Dynamiqs enables rapid simulations and facilitates automatic gradient computation, making it highly suitable for data fitting tasks. This tutorial will guide you through the process of simulating and fitting real-world experimental data, step by step.
+
+## Context of the experiment
+
+!!! note Info
+    This part introduces the kind of data and process that we will fit in the notebook. If you are only interested in the code you can skip it.
+
+!!! example "Diving deep 🤿"
+    You can discover further details about this experiment in the original research paper from which this data was sourced [^1]
```

Comment:
I think this is a bit heavy/insistent: you should definitely cite the paper but probably within the next paragraph.